tiledb/sm/storage_manager/storage_manager.h

```diff
@@ -536,27 +517,6 @@ class StorageManager {
   Status array_get_encryption(
```

Comment:
As you are in here, possibly move to new coding standard?

Reply:
Again, I'd prefer doing this holistically in a separate PR. It will take quite some time moving all functions to the new coding standard.